include/model/CAnomalyScore.h

```diff
@@ -268,13 +268,13 @@ class MODEL_EXPORT CAnomalyScore {
         //! The normalized anomaly score knot points.
         TDoubleDoublePrVec m_NormalizedScoreKnotPoints;
         //! The maximum possible normalized score.
-        double m_MaximumNormalizedScore = 100.0;
+        double m_MaximumNormalizedScore{100.0};
 
         //! The approximate HIGH_PERCENTILE percentile raw score.
         uint32_t m_HighPercentileScore;
```

Comment:
```suggestion
        std::uint32_t m_HighPercentileScore;
```

Reply:
My plan is to try and address some fo the clang tidy suggestions we're getting now we run it over the codebase. As part of this work I'm going to do one big change to switch all our uses of `stdint.h` to `cstdint` and migrate to the std namespace versions. I think I can do this with regex and I reckon it is better than making piecemeal changes for this.